### PR TITLE
Added sounds and background music

### PR DESCRIPTION
**Pull request by carter-greis**

### CODE REVIEW

**Comment by skoolaidl** [comment]
looks good

File: include/GameLogic.h [hunk 1]
```diff
@@ -62,7 +68,7 @@ class GameLogic {
 
 
 	public:
-	GameLogic();
+        GameLogic(){}
```
Comment (jkula25) [inline include/GameLogic.h:71]: Why this change?
